llvm/test/Transforms/InstCombine/icmp-fold-with-cast.ll

```diff
@@ -0,0 +1,175 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 3
+; See PR-70845 for more details
+; RUN: opt < %s -S -passes=instcombine | FileCheck %s
+
+
+define signext i32 @sext_sext(i16 %x, i16 %y) {
```

Comment:
Please drop unused `signext` and `noundef` flags.

Reply:
Generally the tests in this file are not right for InstCombine. We should just directly check the min/max pattern with zext nneg there, nothing more. Preferably directly next to the existing tests for the transform (you can find them by commenting out the transform and seeing which tests break).

Reply:
The technique of commenting and seeing which tests crashed worked really well. Thanks for the trick